dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -102,7 +108,7 @@ object TankGenerator {
         assets
           .combinations(2)
           .map {
-            case List(aa, pa) => if (pa < aa) (aa, pa) else (pa, aa)
+            case List(aa, pa) => if (IssuedAsset(pa.getBytes()).compatId < IssuedAsset(aa.getBytes()).compatId) (aa, pa) else (pa, aa)
```

Comment:
1. Interesting, how it works, I didn't see `AssetPairBuilder.assetIdOrdering` import here.
2. This should not work, because:
```scala
scala> val xs = Array[Byte](1, 2, 3)
     | val ys = java.util.Base64.getEncoder.encodeToString(xs).getBytes
     | java.util.Arrays.equals(ys, xs)
val xs: Array[Byte] = Array(1, 2, 3)
val ys: Array[Byte] = Array(65, 81, 73, 68)
val res1: Boolean = false
```
3. How it should look:
3.1. Use `map` and `Asset.fromString` to parse each asset in `assets`. Throw an exception if it can't decode the asset id. So you will have `List[Asset]`
3.2. Import `AssetPairBuilder.assetIdOrdering` to compare ids.
3.3. Compare ids with `<` as usual
3.4. Use `Asset.toString` to convert `(Asset, Asset)` into `(String, String)`

Reply:
fixed